src/main/java/com/arpnetworking/clusteraggregator/client/AggClientConnection.java

```diff
@@ -55,10 +55,17 @@
      * @param connection Reference to the client connection actor.
      * @param remote The address of the client socket.
      * @param maxConnectionAge The maximum duration to keep a connection open before cycling it.
+     * @param calculateAggregates True to compute cluster aggregations, false to only publish host aggregations
```

Comment:
Is this just cluster aggregations or all aggregations across dimensions? I hope it would control the latter. This is basically for histogram versus no histogram - right?

Reply:
It is the latter.  Technically the aggregations is done on "all dimensions except the host."  Ie, the metric name + dimensions map - host is the key.